constraint/pkg/client/client_opts.go

```diff
@@ -72,3 +79,36 @@ func IgnoreNoReferentialDriverWarning(ignore bool) Opt {
 		return nil
 	}
 }
+
+func EnforcementPoints(eps ...string) Opt {
+	return func(client *Client) error {
+		client.enforcementPoints = eps
+		return nil
+	}
+}
+
+func SetDriverQueryOpt(sourceEP string, opt ...drivers.QueryOpt) {
```

Comment:
This doesn't quite work, since query options should be expressible on a per-call-to-review basis (see alternate design I suggested in a comment on the function signature for Review())